packages/Modal/src/Modal.styles.js

```diff
@@ -80,25 +77,3 @@ export const Modal = styled.div(
     ${stylers.z(1)};
   `
 );
-
-export const Header = styled(OriginalHeader)`
-  flex: none;
-`;
-
-export const ContentWrapper = styled.div`
-  overflow-y: auto;
-
-  &:focus {
-    ${stylers.focusRing.subtle(true)};
-  }
-`;
-
-export const Content = styled(OriginalContent)`
-  ${Header} + ${ContentWrapper} > & {
-    margin-top: -${OriginalContent.SPACE};
```

Comment:
I removed this style. I'm not sure it is necessary, and doesn't look good if you change the background colour of the header for example. The content that is passed into the `Modal.Content` subcomponent should have the correct margins applied (eg margin-top: 0;) http://storybooks.highbond-s3.com/paprika/uxd-812-modal-use-context/?path=/story/messaging-modal-examples--with-header-coloured

Reply:
I think it makes sense to remove this and leave the content styling up to the consumer; but does this make it a breaking change?  Existing modals will look slightly different (the content will be 16px lower).

Reply:
I'm not entirely clear what constitutes a breaking change as this will be a minor visual change, but perhaps? 🤷‍♂️

Reply:
@nahumzs what do you think, is this a breaking change?

Reply:
or @mikrotron, do you think this is style change makes this a breaking change?

Reply:
Yeah, even I find [this topic](https://github.com/semver/semver/issues/161) ambiguous. I don't think semver was created with CSS (or even UI) in mind.   

I guess I would frame it like this: _"Is this detail a part of the API contract that Paprika has with its consumers?"_

If we break the contract, it should be a breaking change.  If the margin of a subcomponent is part of the "contract", then that's a pretty specific and detailed contract! So I'd be inclined to say it is NOT a breaking changed, but should be clearly documented in the CHANGELOG.  

cc @alexzherdev

Reply:
yeah this is tricky and more of a judgement call. You could consider this a bug fix for the component in which case it's fair not to announce this as a breaking change (bug fixes often _are_ breaking changes but you don't make a major bump for every fix).
Since this is a pretty high-traffic component we may want to play it safe and notify consumers they may need a code change via a major bump.

Reply:
Ok In my opinion also this should be considered a breaking change so lets go with that.